src/systems/filecoin_vm/runtime/exitcode/vm_exitcodes.go

```diff
@@ -26,38 +31,45 @@ const (
 	// an actor
 	InvalidMethod
 
-	// InvalidArguments indicates that a method was called with the incorrect
+	// InvalidArgumentsSystem indicates that a method was called with the incorrect
 	// number of arguments, or that its arguments did not satisfy its
 	// preconditions
-	InvalidArguments
+	InvalidArguments_System
 
 	// InsufficientFunds represents a failure to apply a message, as
 	// it did not carry sufficient funds for its application.
-	InsufficientFunds
+	InsufficientFunds_System
 
 	// InvalidCallSeqNum represents a message invocation out of sequence.
 	// This happens when message.CallSeqNum is not exactly actor.CallSeqNum + 1
 	InvalidCallSeqNum
 
-	// OutOfGasError is returned when the execution of an actor method
+	// OutOfGas is returned when the execution of an actor method
 	// (including its subcalls) uses more gas than initially allocated.
 	OutOfGas
 
 	// RuntimeAPIError is returned when an actor method invocation makes a call
 	// to the runtime that does not satisfy its preconditions.
 	RuntimeAPIError
 
-	// MethodPanic is returned when an actor method invocation calls rt.Abort.
-	MethodAbort
+	// RuntimeAssertFailure is returned when an actor method invocation calls
+	// rt.Assert with a false condition.
+	RuntimeAssertFailure
 
 	// MethodSubcallError is returned when an actor method's Send call has
 	// returned with a failure error code (and the Send call did not specify
 	// to ignore errors).
 	MethodSubcallError
 )
 
-var (
-	InvalidSectorPacking = UserDefinedError(1)
+const (
+	InsufficientFunds_User = UserDefinedErrorCode_Placeholder + iota
+	InvalidArguments_User
+	InconsistentState_User
+
+	InvalidSectorPacking
+	SealVerificationFailed
+	DeadlineExceeded
```

Comment:
As mentioned elsewhere, I think that all actor-specified error codes should move into the actors, and the system errors be ideally never referenced from actor code (only from within the VM internals). That can wait for a follow-up though and I'm aware of the APIError problem.